### ISSUE TITLE
Version 6.0.0 causes manifest merger to fail with data binding

### ISSUE THREAD

**Issue by SaeedMasoumi** [JLLeitschuh/ktlint-gradle#134]
I just updated the version from `4.1.0` to `6.0.0`, but I got the below exception while syncing my project: (everything works fine on version 4.1.0)

```
Manifest merger failed : uses-sdk:minSdkVersion 1 cannot be smaller than version 14 declared in library [androidx.databinding:databinding-adapters:3.2.0] /.gradle/caches/transforms-1/files-1.1/databinding-adapters-3.2.0.aar/5499271942aac4ac65cb5ae9c8335a6f/AndroidManifest.xml as the library might be using APIs not available in 1
	Suggestion: use a compatible library with a minSdk of at most 1,
		or increase this project's minSdk version to at least 14,
		or use tools:overrideLibrary="androidx.databinding.library.baseAdapters" to force usage (may lead to runtime failures)
```

**Android Studio Version** : 3.2.0

**build.gradle**
```
plugins {
    id "org.jlleitschuh.gradle.ktlint" version "6.0.0"
}
subprojects {
    apply plugin: "org.jlleitschuh.gradle.ktlint" 
}
```

**Comment by Tapchicoma** [comment]
Can you check if you have the same issue with `5.1.0` version?

Would be nice, if possible, to provide link to project that I can use to reproduce this issue.

**Comment by SaeedMasoumi** [comment]
Yeah, It looks fine with `5.1.0`. 

This the sample project that I create it now, It will throw a similar exception:
```
Manifest merger failed : uses-sdk:minSdkVersion 1 cannot be smaller than version 14 declared in library [com.android.support:appcompat-v7:28.0.0] /.gradle/caches/transforms-1/files-1.1/appcompat-v7-28.0.0.aar/b0906ce2141ef30b2dc05909f08cd634/AndroidManifest.xml as the library might be using APIs not available in 1
	Suggestion: use a compatible library with a minSdk of at most 1,
		or increase this project's minSdk version to at least 14,
		or use tools:overrideLibrary="android.support.v7.appcompat" to force usage (may lead to runtime failures)

```

[ktlint-gradle-sample-6.0.0.zip](https://github.com/JLLeitschuh/ktlint-gradle/files/2430701/ktlint-gradle-sample-6.0.0.zip)

**Comment by Tapchicoma** [comment]
Most probably relates to #131

**Comment by igorwojda** [comment]
Same here 5.1.0 works fine.

**Comment by Tapchicoma** [comment]
Should be fixed in next `6.1.0` version. I want to finish this PR: #136 and then do new release.

**Comment by Tapchicoma** [comment]
Please try released version `6.1.0` and feel free to reopen this issue.

**Comment by igorwojda** [comment]
6.1.0 works fine

Thanks a lot for outstanding support & extremely quick fix
